indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -120,7 +120,43 @@
 }
 
 .section-actions {
-  a + a {
-    margin-left: 3px;
+  a {
+    font-size: 1.2em;
   }
+
+  a + a,
+  a + div {
+    margin-left: 5px;
+  }
+}
+
+.dropdown-menu {
+  display: inline-grid;
+  grid-template-columns: 1fr 1fr;
+
+  & > div:first-child {
+    border-right: 1px solid rgba(34, 36, 38, 0.15);
```

Comment:
can we use a palette color for this?

Reply:
We can, but it might not look the same as the horizontal divider (reused the divider color)

Reply:
fine then, i didn't realize it's a color coming from SUI